types/web3/eth/index.d.ts

```diff
@@ -179,8 +175,8 @@ export default interface Eth {
         cb?: Callback<boolean>
     ): Promise<boolean>;
     sign(
-        address: string,
         dataToSign: string,
+        address: string,
```

Comment:
So the code has a bug. I suggest the following work procedure in such a case:

1. Add a test that fails because of the bug.
2. Fix the bug.
3. Commit (leave the test in!)

So right now please add a regression test for this function.

Reply:
Sure. Will try to create a separate PR for this.